packages/node/test/lib/get-group-id.test.ts

```diff
@@ -1,4 +1,4 @@
-import type { GroupingObject } from '../../src';
```

Comment:
`GroupingObject` doesn't exist in `src/index.ts`!

Reply:
went through and made a bunch of type fixes in https://github.com/readmeio/metrics-sdks/pull/1056